php-fpm/Dockerfile

```diff
@@ -29,9 +29,8 @@ RUN apk add --update \
     make \
     curl
 
-RUN rm -rf /var/cache/apk/* && rm -rf /tmp/*
-
-RUN curl --insecure https://getcomposer.org/composer.phar -o /usr/bin/composer && chmod +x /usr/bin/composer
+RUN rm -rf /var/cache/apk/* && rm -rf /tmp/* && \
+    curl --insecure https://getcomposer.org/composer.phar -o /usr/bin/composer && chmod +x /usr/bin/composer
```

Comment:
Hello, what about using multi-stage build for adding composer into this PHP image?

See: https://medium.com/@othillo/adding-composer-to-php-docker-images-using-multi-stage-builds-2a10967ae6c1